storage/buckets/buckets_test.go

```diff
@@ -278,22 +279,29 @@ func TestLifecycleManagement(t *testing.T) {
 	tc := testutil.SystemTest(t)
 	bucketName := tc.ProjectID + "-storage-buckets-tests"
 
+	ctx := context.Background()
+	testutil.CleanBucket(ctx, t, tc.ProjectID, bucketName)
+
 	if err := enableBucketLifecycleManagement(ioutil.Discard, bucketName); err != nil {
 		t.Errorf("enableBucketLifecycleManagement: %v", err)
 	}
 
 	// verify lifecycle is set
-	m, err := getBucketMetadata(ioutil.Discard, bucketName)
+	client, err := storage.NewClient(ctx)
 	if err != nil {
-		t.Errorf("getBucketMetadata: %#v", err)
+		t.Errorf("storage.NewClient: %v", err)
+	}
+	attrs, err := client.Bucket(bucketName).Attrs(ctx)
+	if err != nil {
+		t.Errorf("Bucket(%q).Attrs: %v", bucketName, err)
```

Comment:
Probably want `t.Fatalf` here and above. No need to continue if there was an error retrieving the attrs.